src/utils.js

```diff
@@ -309,6 +309,31 @@ export const getTypicalChromeExecutablePath = () => {
     }
 };
 
+/**
+ * Wraps the provided Promise with another one that rejects with the given errorMessage
+ * after the given timeoutMillis, unless the original promise resolves or rejects earlier.
+ *
+ * @param {Promise} promise
+ * @param {number} timeoutMillis
+ * @param {string} errorMessage
+ */
+export const addTimeoutToPromise = (promise, timeoutMillis, errorMessage) => {
```

Comment:
Do you want to make this function part of public SDK? I think we should keep it internal, i.e. add @ignore. Also I don't like the name too much but I can't think of anything better now :)

Reply:
Yeah, I forgot that `@ignore`.